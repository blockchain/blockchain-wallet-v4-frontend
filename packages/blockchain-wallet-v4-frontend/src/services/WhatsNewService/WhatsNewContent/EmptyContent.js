import React from 'react'
import styled from 'styled-components'
import { FormattedMessage, FormattedHTMLMessage } from 'react-intl'
import { Text } from 'blockchain-info-components'

const Wrapper = styled.div`
  >div: first-child {
    margin-bottom: 10px;
  }
  a {
    color: ${props => props.theme['brand-secondary']};
  }
`

const EmptyContent = ({ verifyIdentity }) => (
  <Wrapper>
    <Text size='20px'>
      <FormattedMessage
        id='layouts.wallet.header.whatsnew.whatsnew.empty'
        defaultMessage="ðŸŽ‰ You're all caught up!"
      />
    </Text>
    <Text size='12px' weight={300} color='gray-3'>
      <FormattedHTMLMessage
        id='layouts.wallet.header.whatsnew.whatsnew.in_progress'
        defaultMessage="Our team is always working on new features, but if there's something we can improve please let us know about it <a href='https://github.com/blockchain/blockchain-wallet-v4-frontend/issues' rel='noopener noreferrer' target='_blank'>here<a/>."
      />
    </Text>
  </Wrapper>
)
export default EmptyContent
