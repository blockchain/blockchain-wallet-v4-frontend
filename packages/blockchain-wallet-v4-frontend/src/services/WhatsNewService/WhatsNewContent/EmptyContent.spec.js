import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import EmptyContent from './EmptyContent'

jest.mock('blockchain-info-components', () => ({
  Text: 'text'
}))

describe('EmptyContent', () => {
  it('renders correctly', () => {
    const component = shallow(<EmptyContent />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
