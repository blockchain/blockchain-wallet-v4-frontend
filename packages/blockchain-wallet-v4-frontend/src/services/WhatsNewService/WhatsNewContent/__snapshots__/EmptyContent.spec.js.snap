// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyContent renders correctly 1`] = `
<EmptyContent__Wrapper>
  <text
    size="20px"
  >
    <FormattedMessage
      defaultMessage="ðŸŽ‰ You're all caught up!"
      id="layouts.wallet.header.whatsnew.whatsnew.empty"
      values={Object {}}
    />
  </text>
  <text
    color="gray-3"
    size="12px"
    weight={300}
  >
    <FormattedHTMLMessage
      defaultMessage="Our team is always working on new features, but if there's something we can improve please let us know about it <a href='https://github.com/blockchain/blockchain-wallet-v4-frontend/issues' rel='noopener noreferrer' target='_blank'>here<a/>."
      id="layouts.wallet.header.whatsnew.whatsnew.in_progress"
      values={Object {}}
    />
  </text>
</EmptyContent__Wrapper>
`;
